scripts/syntax-check.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# This script is a helper for checking the syntax of any ansible playbook
+
+# Set command and arguments
+cmd=ansible-playbook
+args="--syntax-check $@"
```

Comment:
IIRC syntax check doesn't go down to included files/tasks, I think it will stay just at first playbook level.

We could maybe do a syntax check for all playbook files and then for all task (and is possible also for template/group var/host var) files iteratively to have 100% coverage.

Reply:
you are right, unfortunately you can't call syntax check on the tasks files because they are not full playbooks so they fail. so we can only use it on playbooks. 

I'm hoping that ansible-lint will be able to handle those files.